docs/features/sessions/how-to-ssh-into-a-session.md

```diff
@@ -48,6 +48,14 @@ grid session ssh happy-owl-123
 
 ![](/images/sessions/how-to-ssh-into-session.gif)
 
+:::note
+If trying to use session with ssh and asked for a password: 
+- make sure to do the above Step 0: Create an SSH Key and Step 1: Add the SSH key to Grid
+- make sure that ~/.ssh/config file doesn't have a rule that conflicts with Grid settings
+- Can you make sure that that below line is at the top of the ~/.ssh/config file.
```

Comment:
Change to

"Make sure that the below line is at the top of the ~/.ssh/config file."

Reply:
Thanks made that change